docs/howto/how-to-make-api-tool/api-tool-guide.md

```diff
@@ -0,0 +1,250 @@
+[Workspace initialization] .  Initialize backend
+[Workspace initialization] Creating ~\sdk-api-tool\backend\ayx_plugins
+[Workspace initialization] Creating file ~\sdk-api-tool\backend\requirements-local.txt
+[Workspace initialization] Creating file ~\sdk-api-tool\backend\requirements-thirdparty.txt
+[Workspace initialization] Creating file ~\sdk-api-tool\backend\setup.py
+[Workspace initialization] Creating file ~\sdk-api-tool\backend\ayx_plugins\__init__.py
+[Workspace initialization] .  Create tests directory
+[Workspace initialization] .  Initialize UI
+[Workspace initialization] finished
+Created Alteryx workspace in directory: ~\sdk-api-tool
+Workspace settings can be modified in: ayx_workspace.json
+[Generating config files] started
+[Generating config files] .  generate_config_files:generate_config_xml
+[Generating config files] Generating top level config XML file...
+[Generating config files] finished
+```
+
+### 2. Creating a plugin
+The next step is to add a plugin to the workspace. We do this using the `create-ayx-plugin` command. Fill out the prompts and then you will have the template code for your SDK Plugin. For this tool, we are choosing the `Input` tool type.
+
+```
+$ ayx_plugin_cli create-ayx-plugin
+Tool Name: API tool
+Tool Type (input, multiple-inputs, multiple-outputs, optional, output, single-input-single-output, multi-connection-input-anchor) [single-input-single-output]: input
+Description []: My API Tool
+Tool Version [1.0]: 1.0
+DCM Namespace []:
+Creating input plugin: API tool
+[Create plugin] started
+[Create plugin] Downloading UI components via git
+[Create plugin] Cloning into '.ayx_cli.cache\ui_tool_template'...
+[Create plugin] .  Create plugin
+[Create plugin] Installing UI components via npm
+[Create plugin] Creating Alteryx Plugin...
+[Create plugin] Copying example tool to ~\sdk-api-tool\backend\ayx_plugins...
+[Create plugin] Added new tool to package directory: ~\sdk-api-tool\backend\ayx_plugins\a_p_i_tool.py
+[Create plugin] finished
+[Generating config files] started
+[Generating config files] .  generate_config_files:generate_config_xml
+[Generating config files] Generating top level config XML file...
+[Generating config files] .  generate_config_files:generate_tool_config_xml
+[Generating config files] Generating tool configuration XMLs...
+[Generating config files] Generating APItool XML...
+[Generating config files] Done!
+[Generating config files] .  generate_config_files:generate_manifest_jsons
+[Generating config files] Generating manifest.json for APItool...
+[Generating config files] Done!
+[Generating config files] finished
+[Generating test files for APItool] started
+[Generating test files for APItool] .  Generate tests
+[Generating test files for APItool] finished
+```
+
+After this command finishes, you will see a file named `apitool.py` under `~/backend/ayx_plugins/` with the boilerplate code. Upon opening the file, you should see something like
```

Comment:
TW review - update to:

After this command finishes, you will see a file named `apitool.py` under `~/backend/ayx_plugins/` with the boilerplate code. When you open the file, you should see something like this:

Reply:
o wow, finally the future tense `will` is okay here?? :P

Reply:
haha, ya it's fine here since we are talking about something that happens after something else.